sympy/solvers/ode/tests/ode_test_suite/test_kamke.py

```diff
@@ -0,0 +1,413 @@
+        "kamke_1.300": 6*x*y(x)**2*Derivative(y(x), x) + x + 2*y(x)**3
+    }
+
+    def get_example(self, example):
+        return self.eqs[example]
+
+    def test_example(self, example, hint="default", verify=True, dsolve_timelim=10, checkodesol_timelim=10, log=False):
+        start = time.time()
+        eq = self.get_example(example)
+        sol = None
+        output = example + " "
+
+        try:
+            # Try to find the solution to the equation
+            with time_limit(dsolve_timelim):
+                sol = dsolve(eq, y(x), hint)
+                if isinstance(sol, Eq):
+                    sol = [sol]
+        except (TimeOutError, ValueError, NotImplementedError, TypeError):
+            # Solution not found / timeout
+            output += "dsolve failed"
+
+        # If a solution is found
+        if sol is not None and verify:
+            try:
+                # Try to verify if the solution is correct
+                assert len(sol)
+                with time_limit(checkodesol_timelim):
+                    checks = checkodesol(eq, sol, y(x))
+                assert any([x[0] for x in checks])
+                output += "solved"
+            except AssertionError:
+                # Wrong solution
+                output += "dsolve failed"
+            except (TimeOutError, ValueError, NotImplementedError, TypeError):
+                # Checkodesol unable to verify / timeout
+                output += "checkodesol failed"
+        output += f" in {time.time() - start} seconds\n"
+        returns = sol
+        if log:
+            returns = [sol, output]
+        return returns
+
+
+    def test_all_examples(self, hint="default", verify=True, dsolve_timelim=10, checkodesol_timelim=10, report_path=None):
+        if report_path is None:
+            report_path = "test_report.txt"
+
+        start = time.time()
+        # Counts for result
+        solved = 0
+        check_slow = 0
+        fail = 0
+
+        with open(report_path, "w") as output_file:
+            for example in self.eqs:
+                # Process is not terminating despite using
+                # time limit. Checkodesol is too slow for
+                # these cases. This should be fixed.
+                if example in ["kamke_1.14", "kamke_1.24", "kamke_1.31", "kamke_1.47", \
+                "kamke_1.48", "kamke_1.94", "kamke_1.113", "kamke_1.205"]:
+                    continue
+
+                sol, output = self.test_example(example, hint, verify, dsolve_timelim, checkodesol_timelim, log=True)
+                output_file.write(output)
+                output_file.flush()
+
+            # Summary
+            output_file.write("Total time taken" + str(time.time() - start))
+            output_file.write("No. of ODEs solved:" + str(solved))
+            output_file.write("No. of ODEs failed:" + str(fail))
+            output_file.write("No. of ODEs solved, but not checked:" + str(check_slow))
+            output_file.close()
```

Comment:
At the end of the file you can put 

```py
if __name__ == '__main__':
    test_all_examples()
```

and you will be able to run it as a script (`python -m sympy.solvers.ode.tests.test_kamke`). You could also put a little argparse parsing in a function to add the options are command line arguments if you wanted.